### PR TITLE
HM_dao5

### PR DESCRIPTION
**Pull request by grom4er**
Create, fill and test with jdbs DAO

### CODE REVIEW

File: src/main/java/taxiservice/dao/jbdc/ManufacturerJdbcDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        String query = "SELECT * FROM manufacturers "
+                + "WHERE manufacturer_id = ? AND deleted = false";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query);
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.ofNullable(getManufacturer(resultSet));
+            } else {
+                throw new RuntimeException("");
```
Comment (Ketch92) [inline src/main/java/taxiservice/dao/jbdc/ManufacturerJdbcDaoImpl.java:52]: you can move this throw to the very end of the method and remove else part

File: src/main/java/taxiservice/util/ConnectionUtil.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    static {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            // TODO
+        }
```
Comment (Ketch92) [inline src/main/java/taxiservice/util/ConnectionUtil.java:14]: why is it empty?

**Review by Ketch92 — CHANGES_REQUESTED** [review]

File: src/main/java/taxiservice/dao/jbdc/ManufacturerJdbcDaoImpl.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+import taxiservice.models.Manufacturer;
+import taxiservice.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerJdbcDaoImpl implements ManufacturerDao {
+
+    @Override
+    public Manufacturer create(Manufacturer element) {
+        String query = "INSERT INTO manufacturers "
+                + "(manufacturer_name, manufacturer_country) VALUE (?, ?)";
```
Comment (boroda4436) [inline src/main/java/taxiservice/dao/jbdc/ManufacturerJdbcDaoImpl.java:23]:
```suggestion
                + "(name, country) VALUE (?, ?)";
```
you already know that this is manufacturer

**Review by boroda4436 — CHANGES_REQUESTED** [review]

File: src/main/java/taxiservice/resources/initDB.sql [hunk 4]
```diff
@@ -0,0 +1,10 @@
+CREATE DATABASE `Taxi_service`;
```
Comment (boroda4436) [inline src/main/java/taxiservice/resources/initDB.sql:1]:
```suggestion
CREATE DATABASE `taxi_service`;
```

**Review by boroda4436 — APPROVED** [review]

**Pull request merged**